api/python/cellxgene_census/pyproject.toml

```diff
@@ -31,7 +31,7 @@ dependencies= [
     # NOTE: the tiledbsoma version must be >= to the version used in the Census builder, to
     # ensure that the assets are readable (tiledbsoma supports backward compatible reading).
     # Make sure this version does not fall behind the builder's tiledbsoma version.
-    "tiledbsoma==1.4.3",
+    "tiledbsoma~=1.4.0",
```

Comment:
given that we are _already_ using 1.4.3, is there a reason you reverted back to 1.4.0?

Recommend:
> tiledbsoma~=1.4.3